src/files/source/metadata.php

```diff
@@ -1009,6 +1024,18 @@
     'deny:server.read' => [
         'description' => 'Prohibits reading of the server',
     ],
+    'deny:server.read-all' => [
+        'description' => 'Prohibits reading of extended data fo the server',
+    ],
+    'deny:server.read-billing' => [
+        'description' => 'Prohibits reading of tariff and sale information of server',
+    ],
+    'deny:server.wizzard-read' => [
```

Comment:
```suggestion
    'deny:server.read-wizzard' => [
```

For other permissions, `read` is a prefix.